Tests/SearchForResults.cs

```diff
@@ -308,4 +308,51 @@ private async Task SetupTuitionPartner()
         });
     }
+
+    [Fact]
+    public async Task Test_Search_Results_Count()
+    {
+        const int numberOfTuitionPartners = 55;
+        await Fixture.ExecuteDbContextAsync(async db =>
+        {
+            for (int tuitionPartnersAdded = 1; tuitionPartnersAdded < numberOfTuitionPartners; tuitionPartnersAdded++)
+            {
+                db.TuitionPartners.Add(new Domain.TuitionPartner
+                {
+                    Id = tuitionPartnersAdded,
+                    SeoUrl = "a-tuition-partner" + tuitionPartnersAdded.ToString(),
+                    Name = "A Tuition Partner" + tuitionPartnersAdded.ToString(),
+                    Website = "https://a-tuition-partner.testdata/ntp " + tuitionPartnersAdded.ToString(),
+                    Description = "A Tuition Partner Description",
+                    PhoneNumber = "0123456789",
+                    Email = "ntp@a-tuition-partner.testdata",
+                    HasSenProvision = false,
+                    LocalAuthorityDistrictCoverage = new List<LocalAuthorityDistrictCoverage>
+                {
+                    new() { LocalAuthorityDistrictId = 1, TuitionTypeId = (int)TuitionTypes.InSchool },
+                    new() { LocalAuthorityDistrictId = 2, TuitionTypeId = (int)TuitionTypes.InSchool },
+                    new() { LocalAuthorityDistrictId = 2, TuitionTypeId = (int)TuitionTypes.Online   },
+                    new() { LocalAuthorityDistrictId = 9, TuitionTypeId = (int)TuitionTypes.Online   },
+                },
+                    SubjectCoverage = new List<SubjectCoverage>
+                {
+                    new() { TuitionTypeId = (int)TuitionTypes.InSchool, SubjectId = Subjects.Id.KeyStage3English },
+                    new() { TuitionTypeId = (int)TuitionTypes.InSchool, SubjectId = Subjects.Id.KeyStage3Maths },
+                },
+                    Prices = new List<Price>
+                {
+                    new() { TuitionTypeId = (int)TuitionTypes.InSchool, SubjectId = Subjects.Id.KeyStage3English, GroupSize = 2, HourlyRate = 12.34m },
+                    new() { TuitionTypeId = (int)TuitionTypes.InSchool, SubjectId = Subjects.Id.KeyStage3English, GroupSize = 3, HourlyRate = 12.34m },
+                    new() { TuitionTypeId = (int)TuitionTypes.InSchool, SubjectId = Subjects.Id.KeyStage3Maths, GroupSize = 2, HourlyRate = 56.78m },
+                    new() { TuitionTypeId = (int)TuitionTypes.InSchool, SubjectId = Subjects.Id.KeyStage3Maths, GroupSize = 3, HourlyRate = 56.78m },
+                    new() { TuitionTypeId = (int)TuitionTypes.Online, SubjectId = Subjects.Id.KeyStage3Maths, GroupSize = 3, HourlyRate = 56.78m },
+                }
+                });
+            }
+            await db.SaveChangesAsync();
+        });
+        var result = await Fixture.SendAsync(new SearchResults.Query());
+        result!.Results.Should().NotBeNull();
+        result!.Results!.Count.Should().Be(54);
```

Comment:
`numberOfTuitionPartners` is `55`, I think it would be good if we could change the loop so that it added the number of partners specified, and then we could use `numberOfTuitionPartners` in the assertion.  Otherwise people coming to read this in the future will wonder why the search results returns one less than it looks.

```suggestion
        result!.Results!.Count.Should().Be(numberOfTuitionPartners);
```

Reply:
Changed